app/server/ruby/lib/sonicpi/synths/synthinfo.rb

```diff
@@ -3290,6 +3290,214 @@ def arg_defaults
       end
     end
 
+    class WinwoodLead < SonicPiSynth
+      def name
+        "Winwood Lead"
+      end
+
+      def introduced
+        Version.new(3,3,0)
+      end
+
+      def synth_name
+        "winwood_lead"
+      end
+
+      def on_start(studio, args_h)
+        args_h[:rand_buf] = studio.rand_buf_id
+      end
+
+      def doc
+        "A lead synth inspired by the Winwood songs from the early 80s.  Adapted for Sonic Pi from [Steal This Sound](https://raw.githubusercontent.com/supercollider/supercollider/develop/examples/demonstrations/stealthissound.scd). Published there under [GPL v3](https://www.gnu.org/licenses/gpl-3.0.en.html), so re-published under the same terms. The source code is available [on the Sonic Pi GitHub repository](https://github.com/sonic-pi-net/sonic-pi). Date of modification: 10.01.2021"
+      end
+
+      def arg_defaults
+        {
+          :note => 69,
+          :note_slide => 0,
+          :note_slide_shape => 1,
+          :note_slide_curve => 0,
+          :amp => 1,
+          :amp_slide => 0,
+          :amp_slide_shape => 1,
+          :amp_slide_curve => 0,
+          :pan => 0,
+          :pan_slide => 0,
+          :pan_slide_shape => 1,
+          :pan_slide_curve => 0,
+
+          :attack => 0.01,
+          :decay => 0,
+          :sustain => 0.9,
+          :release => 0.05,
+          :attack_level => 1,
+          :decay_level => 0.5,
+          :sustain_level => 0.5,
+
+          :cutoff => 119,
+          :cutoff_slide => 0,
+          :cutoff_slide_shape => 1,
+          :cutoff_slide_curve => 0,
+
+          :res => 0.8,
+          :res_slide => 0,
+          :res_slide_shape => 1,
+          :res_slide_curve => 0,
+          :lfo_width => 0.01,
+          :lfo_width_slide => 0,
+          :lfo_width_slide_shape => 1,
+          :lfo_width_slide_curve => 0,
+          :lfo_rate => 8,
+          :lfo_rate_slide => 0,
+          :lfo_rate_slide_shape => 1,
+          :lfo_rate_slide_curve => 0,
+          :seed => 0,
+        }
+      end
+      
+      def specific_arg_info
+        {
+          :seed =>
+          {
+            :doc => "Seed value for rand num generator used for the phase offset of the triangle low-frequency oscillator (LFO)",
+            :modulatable => false
+          },
+          :lfo_width =>
+          {
+            :doc => "Width of the low-frequency oscillator (LFO) which determines how wide base tones oscillate around their base frequencies",
+            :modulatable => true
```

Comment:
Does `lfo_width:` need validations? (same with other non-default opts in these synths)

Reply:
Oh, yes, thanks, both ``lfo_rate``and ``lfo_width`` should be non-negative:

```
:validations => [v_positive(:lfo_rate)],
```
and 

```
:validations => [v_positive(:lfo_width)],
```
in their corresponding sections.

Reply:
Agreed for `:lfo_width`, but I think `:lfo_rate` should be greater than zero...

Reply:
Just checked that in SuperCollider: ``:lfo_rate`` is fed into an ``LFTri``oscillator as the frequency argument. Zero is OK and doesn't break it. It gives a flat line, of course. For practical reasons, I'd allow zero. This way, for example, the user can slide down the rate to zero without causing a validation error.